### PR TITLE
Fixed translate categories script.

### PR DESCRIPTION
**Pull request by deathbaba**

### CODE REVIEW

File: tools/unix/translate_categories.sh [hunk 1]
```diff
@@ -1,5 +1,17 @@
 #!/bin/bash
 
-DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
+set -e -u
 
-${DIR}/translate.awk {en=en+ru+uk+de+fr+it+es+ko+ja+cs+nl+zh-TW+pl+pt+hu+th+zh-CN+ar+da+tr+sv} "$1"
+SRC=en
+WORD=$1
+if [[ "$#" -gt 2 ]]; then
+  SRC=$1
```
Comment (ygorshenin) [inline tools/unix/translate_categories.sh]:
```bash
SRC="$1"
WORD="$2"
```
Вообще если параметров больше чем 2 то мб следует показывать usage?

File: tools/unix/translate_categories.sh [hunk 2]
```diff
@@ -1,5 +1,17 @@
 #!/bin/bash
 
-DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
+set -e -u
 
-${DIR}/translate.awk {en=en+ru+uk+de+fr+it+es+ko+ja+cs+nl+zh-TW+pl+pt+hu+th+zh-CN+ar+da+tr+sv} "$1"
+SRC=en
+WORD=$1
+if [[ "$#" -gt 2 ]]; then
```
Comment (ygorshenin) [inline tools/unix/translate_categories.sh]: No need in outer square brackets.

Comment (deathbaba) [inline tools/unix/translate_categories.sh]: I prefer to always use double brackets - they are more powerful and there is a less chance for error. I can change this case though.

File: tools/unix/translate_categories.sh (hunk 1 above)
Comment (deathbaba) [inline tools/unix/translate_categories.sh]: ok

**Comment by deathbaba** [comment]
@ygorshenin PTAL please.

File: tools/unix/translate_categories.sh [hunk 3]
```diff
@@ -1,5 +1,24 @@
 #!/bin/bash
 
-DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
+set -e -u
 
-${DIR}/translate.awk {en=en+ru+uk+de+fr+it+es+ko+ja+cs+nl+zh-TW+pl+pt+hu+th+zh-CN+ar+da+tr+sv} "$1"
+if [ $# -eq 0 ]; then
```
Comment (ygorshenin) [inline tools/unix/translate_categories.sh]:
```bash

case $# in
  1) SRC=en
       WORD="$1"
       ;;
  2) SRC="$1"
      WORD="$2"
      ;;
  *) echo "Usage ..."
      exit 1
      ;;
esac

...
```

**Comment by deathbaba** [comment]
Fixed. @ygorshenin please merge.

**Comment by ygorshenin** [comment]
LGTM

**Pull request merged**